packages/ai-core/src/application-api.ts

```diff
@@ -24,106 +24,135 @@ export const ApplicationApi = {
    * Return all Argo CD application data objects.
    *
    * @param queryParameters - Object containing the following keys: $top, $skip, $count.
+   * @param headerParameters - Object containing the following keys: Authorization.
    * @returns The request builder, use the `execute()` method to trigger the request.
    */
-  kubesubmitV4ApplicationsGetAll: (queryParameters?: {
-    $top?: number;
-    $skip?: number;
-    $count?: boolean;
-  }) =>
+  kubesubmitV4ApplicationsGetAll: (
+    queryParameters?: { $top?: number; $skip?: number; $count?: boolean },
+    headerParameters?: { Authorization?: string }
```

Comment:
[q] Why is the `Authorization` header only added to a few API, while all others lack it. I assume every request would need this header.
If we supply this header via the destination instead in some API for e.g Artifact, why can't the same be followed here?

Reply:
Since we only added support for in header parameters, these were already in the spec but are only visible now. Its a good point. I assume we would just supply this via the service binding. So the `Authorization` headers can be removed @jjtang1985 ?

Reply:
@shibeshduw , as long as this is generated, it's ok, because the generated client should reflect what's defined in the spec.
I double check the Java client and the spec, where the `Authorization` header is shown in both cases, under the application api.